packages/manager/modules/vps/src/migration/components/plan/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,17 @@
+{
+  "vps_common_cancel": "Annuler",
+  "vps_common_confirm": "Confirmer",
+  "vps_migrate_schedule_title": "Migration de: {{vpsName}}",
+  "vps_migrate_schedule_description": "Migration description fadf fadf fadf fad fafa dasfafa fadsfad",
```

Comment:
???

Reply:
Has to be updated once we get the content :D